include/aoapplication.h

```diff
@@ -342,8 +410,12 @@ class AOApplication : public QApplication
   // Returns p_char's gender
   QString get_gender(QString p_char);
 
-  // Get the location of p_image, which is either in a theme variant subfolder,
-  // a theme folder, or default theme folder
+  // Get the location of p_image, which is either in a
```

Comment:
These comments could also be made into proper documentation.

Reply:
See #dev